collaboration/service/v1alpha/user_group.proto

```diff
@@ -49,6 +49,8 @@ message SaveUserGroupRequest {
   google.protobuf.StringValue name = 2;
   google.protobuf.StringValue organization = 3;
   io.spoud.sdm.global.mutation.v1.PropertyMap properties = 4;
+  google.protobuf.StringValue label = 5;
```

Comment:
I would honestly use the properties. And I think we have already email in there, no ?

Reply:
It's a field on all the other entities like TechUser etc. so I think we should keep it here.